eventespresso/barista
# UI adjustment in REM

Fixes [#816](https://github.com/eventespresso/cafe/issues/816)

### testing

The process is a little different for JS code in the Barista repo, you'll need to do the following:

- in a terminal navigate to your cafe repo's plugin folder

- then clone this repo using
   ```
   git clone git@github.com:eventespresso/barista.git
   ```

- switch into that folder
   ```
   cd barista
   ```

- run the following commands
   ```
   git checkout FIX/ui-adjustment-in-rem
   yarn build
   // if you don't have yarn installed, then contact brent for help
   ```

- then go to your browser and log into your cafe test site (the one you just cloned the barista repo into)

- go to the WordPress Plugins admin page

- find and activate the "Barista for Event Espresso" plugin (NOT the "Barista Tester" plugin)

- now go to the Event Espresso Events admin page and add/edit an event

- follow the steps from your original post to recreate the reported issue 

- [ ] confirm that the REM UI issues have been fixed
